### PR TITLE
DL-13464 Clean logback file

### PR DESCRIPTION
**Pull request by jmountain-hmrc**

### CODE REVIEW

**Comment by nadinJ** [comment]
PR bot comments not addressed

File: conf/logback-test.xml [hunk 1]
```diff
@@ -14,34 +14,13 @@
...
-        <appender-ref ref="ACCESS_LOG_FILE" />
-    </logger>
-
-    <logger name="com.ning.http.client.providers.netty" additivity="false">
-        <appender-ref ref="STDOUT_IGNORE_NETTY" />
-    </logger>
-
     <logger name="com.google.inject" level="OFF"/>
 
     <logger name="org.asynchttpclient.netty" level="OFF"/>
```
Comment (paulanderson1234) [inline conf/logback-test.xml:26]: These entries relating to nety can also be removed:
logger name="org.asynchttpclient.netty" level="OFF"/
logger name="io.netty.buffer" level="OFF"/
logger name="play.core.netty" level="OFF"/

File: conf/logback.xml [hunk 2]
```diff
@@ -14,43 +14,21 @@
...
-    </logger>
-
-    <logger name="com.ning.http.client.providers.netty" additivity="false">
-        <appender-ref ref="STDOUT_IGNORE_NETTY" />
-    </logger>
-
     <logger name="uk.gov" level="INFO"/>
 
     <logger name="com.google.inject" level="INFO"/>
     <logger name="org.asynchttpclient.netty" level="INFO"/>
```
Comment (paulanderson1234) [inline conf/logback.xml:27]: These entries relating to netty can also be removed:
logger name="org.asynchttpclient.netty" level="INFO"/
logger name="play.core.netty.utils.ServerCookieDecoder" level="INFO"/
logger name="io.netty.util.internal" level="INFO"

File: conf/logback-test.xml (hunk 1 above)
Comment (paulanderson1234) [reply conf/logback-test.xml]: Removed

File: conf/logback.xml (hunk 2 above)
Comment (paulanderson1234) [reply conf/logback.xml]: Removed

**Review by paulanderson1234 — APPROVED** [review]

**Pull request merged**